### DISCUSSION TITLE
Custom alerts in detections

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#13383] [2.4]
### Version

2.4.80

### Installation Method

Network installation on Ubuntu

### Description

configuration

### Installation Type

Distributed

### Location

other (please provide detail below)

### Hardware Specs

other (please provide detail below)

### CPU

6

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hello,
It's possible to manage custom elastalert rules via detections that uses other elastalert output modules, like slack or custom modules? OR this tool can manage only modules.so.securityonion-es.SecurityOnionESAlerter?
Thanks.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TOoSmOotH** [comment]
Check: https://github.com/Security-Onion-Solutions/securityonion/discussions/13380

For elastalert you will have to manage it through the command line like in previous versions.
